konfera/utils.py

```diff
@@ -41,3 +49,48 @@ def update_event_context(event, context, show_sponsors=True):
 
 def currency_round_up(money):
     return money.quantize(Decimal('1.00'), rounding=ROUND_UP)
+
+
+def validate_email_template(raw_template, input):
+    required_keys = set(re.findall('{(.+?)}', raw_template))
+    if required_keys.issubset(set(input.keys())):
+        return raw_template.format(**input)
+    return
+
+
+class EmailTemplateError(Exception):
+    pass
+
+
+def send_email(addresses, subject, template, input=None, verbose=0):
```

Comment:
input is built in function, please consider different name.

we are passing also the content of the templates, how about we just pass the name (that should be called in the DB) and we will do a search in the DB here. You can have separete search_template function that will raise error if not found or return both text and html template if exists in DB. I can imagine it would clear the code elsewhere. Just a thing to consider, havent look at the place where we do call them, this is just a thought.